packages/features/ee/platform/components/CreateANewPlatformForm.tsx

```diff
@@ -65,11 +65,13 @@ const CreateANewPlatformFormChild = ({ session }: { session: Ensure<SessionConte
     onError: (err) => {
       if (err.message === "organization_url_taken") {
         newOrganizationFormMethods.setError("slug", { type: "custom", message: t("url_taken") });
+        setServerErrorMessage(err.message);
```

Comment:
Don't know why we were not displaying error messages in case the request fails so bringing those back.